ui/lib/config-ui/addon/components/login-settings/page/details.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+
+/**
+ * @module Page::LoginSettingsRuleDetails
+ * Page::LoginSettingsRuleDetails components are used to display list of rules.
+ * @example
+ * ```js
+ * <Page::LoginSettingsRuleDetails @rule={{this.rule}}  />
+ * ```
+ * @param {object} rule - rule object
```

Comment:
More here as well. What is the rule object, where is it coming from. Is the default empty? [Here](https://github.com/hashicorp/vault/blob/main/ui/app/components/database-role-setting-form.js#L21) is a good example of an object param description.